README.md

```
@@ -0,0 +1,140 @@
+制限時間内に VR 側が規定数のモンスターを倒せばVR側の勝利、タイムアップになればスマートフォン側の勝利となります。
+
+**関連リンク:**
+- [解説ページ (Topaz)](https://topaz.dev/projects/9ae51df3f2987dd2d0b5)
+- [配布ページ (itch.io)](https://kou050223.itch.io/chebu-warts)
+- [設計ドキュメント (Figma)](https://www.figma.com/board/tXrwssYSDdPiM16LxjsQ8G/)
+
+---
+
+## ゲームの流れ
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)